pkgs/development/python-modules/maestral/default.nix

```diff
@@ -65,6 +66,14 @@ buildPythonPackage rec {
     "--prefix PYTHONPATH : $out/${python.sitePackages}"
   ];
 
+  patches = [
+    # Allow the use of survey >= 5 ; remove after new maestral release
+    (fetchpatch {
+      url = "https://github.com/samschott/maestral/commit/2c50d2ddb49a845ea97bd6b0f68c45d723fb304c.patch";
+      hash = "sha256-1YvnnTAJPfj3Zt8K/J4GMhLsE2KhJsxP/j4DblF4LwI=";
+    })
+  ];
```

Comment:
We should probably use the pythonRelaxDepsHook hook here instead

Reply:
Sure, whatever gets this fixed on master the fastest.